wrappers/ortools.rb

```diff
@@ -587,6 +587,7 @@ def parse_output(vrp, _services, points, _matrix_indices, _cost, _iterations, ou
                   current_matrix_index = point.matrix_index
                   route_data = build_route_data(vehicle_matrix, previous_matrix_index, current_matrix_index)
                   current_activity = {
+                    original_service_id: service.original_id,
```

Comment:
Currently It cannot be used if we are not in a periodic context ? It will return a nil value